tests/backends/sqlite/tests.py

```diff
@@ -19,6 +23,14 @@
 class Tests(TestCase):
     longMessage = True
 
+    def test_min_version(self):
+        msg = 'SQLite 3.8.3 or later is required (found 3.8.2).'
+        with mock.patch.object(dbapi2, 'sqlite_version_info', (3, 8, 2)), \
+                mock.patch.object(dbapi2, 'sqlite_version', '3.8.2'), \
+                self.assertRaisesMessage(ImproperlyConfigured, msg):
+            import django.db.backends.sqlite3.base
```

Comment:
Might want to add a comment mentioning this is to make sure the module is loaded even when tests are not run against SQLite before calling `reload` below.